cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -163,6 +166,9 @@ func (i *deployCommand) InitHook() {
 	i.command.Flags().StringVar(&i.flags.imageVersion, "image-version", "", "Image version for standard Kogito build images. Ignored if a custom image is set for image-s2i or image-runtime.")
 	i.command.Flags().StringVar(&i.flags.mavenMirrorURL, "maven-mirror-url", "", "Internal Maven Mirror to be used during source-to-image builds to considerably increase build speed, e.g: https://my.internal.nexus/content/group/public")
 	i.command.Flags().BoolVar(&i.flags.enableIstio, "enable-istio", false, "Enable Istio integration by annotating the Kogito service pods with the right value for Istio controller to inject sidecars on it. Defaults to false")
+	i.command.Flags().StringVar(&i.flags.projectGroupId, "project-group-id", "com.company", "Indicates the unique identifier of the organization or group that created the project.")
```

Comment:
Shouldn't we have empty values by default like it was before ?

Reply:
set default value as empty